test/data/repositories/rhel-84.json

```diff
@@ -0,0 +1,109 @@
+{
+  "aarch64": [
+    {
+      "name": "baseos",
+      "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8.4/aarch64/baseos/os",
```

Comment:
Adding this non-eus repo is not needed. All the content from 8.4 is available in EUS.